packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/ground_overlay.dart

```diff
@@ -0,0 +1,385 @@
+  ///     createLocalImageConfiguration(context),
+  ///     'assets/images/ground_overlay.png',
+  ///     bitmapScaling: MapBitmapScaling.none,
+  ///   ),
+  ///   bounds: LatLngBounds(
+  ///     southwest: LatLng(37.42, -122.08),
+  ///     northeast: LatLng(37.43, -122.09),
+  ///   ),
+  /// );
+  factory GroundOverlay.fromBounds({
+    required GroundOverlayId groundOverlayId,
+    required MapBitmap image,
+    required LatLngBounds bounds,
+    Offset anchor = const Offset(0.5, 0.5),
+    double bearing = 0.0,
+    double transparency = 0.0,
+    int zIndex = 0,
+    bool visible = true,
+    bool clickable = true,
+    VoidCallback? onTap,
+  }) {
+    return GroundOverlay._(
+      groundOverlayId: groundOverlayId,
+      image: image,
+      bounds: bounds,
+      anchor: anchor,
+      bearing: bearing,
+      transparency: transparency,
+      zIndex: zIndex,
+      visible: visible,
+      clickable: clickable,
+      onTap: onTap,
+    );
+  }
+
+  /// Creates a [GroundOverlay] to given [position] with the given [image].
+  ///
+  /// Example:
+  /// ```dart
+  /// GroundOverlay.fromPosition(
+  ///   groundOverlayId: const GroundOverlayId('overlay_id'),
+  ///   image: await AssetMapBitmap.create(
+  ///     createLocalImageConfiguration(context),
+  ///     'assets/images/ground_overlay.png',
+  ///     bitmapScaling: MapBitmapScaling.none,
+  ///   ),
+  ///   position: LatLng(37.42, -122.08),
+  ///   width: 100,
+  ///   height: 100,
+  ///   zoomLevel: 14,
+  /// );
+  /// ```
+  factory GroundOverlay.fromPosition({
+    required GroundOverlayId groundOverlayId,
+    required MapBitmap image,
+    required LatLng position,
+    double? width,
+    double? height,
+    Offset anchor = const Offset(0.5, 0.5),
+    double bearing = 0.0,
+    double transparency = 0.0,
+    int zIndex = 0,
+    bool visible = true,
+    bool clickable = true,
+    VoidCallback? onTap,
+    double? zoomLevel,
+  }) {
+    return GroundOverlay._(
+      groundOverlayId: groundOverlayId,
+      image: image,
+      position: position,
+      width: width,
+      height: height,
+      anchor: anchor,
+      bearing: bearing,
+      transparency: transparency,
+      zIndex: zIndex,
+      visible: visible,
+      clickable: clickable,
+      onTap: onTap,
+      zoomLevel: zoomLevel,
+    );
+  }
+
+  /// Uniquely identifies a [GroundOverlay].
+  final GroundOverlayId groundOverlayId;
+
+  @override
+  GroundOverlayId get mapsId => groundOverlayId;
+
+  /// A description of the bitmap used to draw the ground overlay.
+  ///
+  /// To create ground overlay from assets, use [AssetMapBitmap],
+  /// [AssetMapBitmap.create] or [BitmapDescriptor.asset].
+  ///
+  /// To create ground overlay from raw PNG data use [BytesMapBitmap]
+  /// or [BitmapDescriptor.bytes].
+  ///
+  /// [MapBitmap.bitmapScaling] must be set to [MapBitmapScaling.none].
+  final MapBitmap image;
+
+  /// Geographical location to which the anchor will be fixed. The relative
```

Comment:
Please sweep all of these comments to have them adhere to https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph